R/derive_var_anrind.R

```diff
@@ -29,47 +38,72 @@
 #' library(admiral.test)
 #' data(admiral_vs)
 #'
-#' ref_ranges <- tribble(
+#' ref_ranges1 <- tribble(
 #'   ~PARAMCD, ~ANRLO, ~ANRHI, ~A1LO, ~A1HI,
 #'   "DIABP",      60,     80,    40,    90,
 #'   "PULSE",      60,    100,    40,   110
 #' )
 #'
-#' admiral_vs %>%
+#' ref_ranges2 <- tribble(
+#'   ~PARAMCD, ~ANRLO, ~ANRHI,
+#'   "DIABP",      60,     80,
+#'   "PULSE",      60,    100
+#' )
+#'
+#' vs <- admiral_vs %>%
 #'   mutate(
 #'     PARAMCD = VSTESTCD,
 #'     AVAL = VSSTRESN
 #'   ) %>%
-#'   filter(PARAMCD %in% c("PULSE", "DIABP")) %>%
-#'   derive_vars_merged(ref_ranges, by_vars = exprs(PARAMCD)) %>%
-#'   derive_var_anrind() %>%
+#'   filter(PARAMCD %in% c("PULSE", "DIABP"))
+#'
+#' vs1 <- vs %>%
+#'   derive_vars_merged(ref_ranges1, by_vars = exprs(PARAMCD))
+#'
+#' vs2 <- vs %>%
+#'   derive_vars_merged(ref_ranges2, by_vars = exprs(PARAMCD))
+#'
+#' vs_anrind1 <- vs1
+#' derive_var_anrind(use_a1hia1lo = TRUE) %>%
+#'   select(USUBJID, PARAMCD, AVAL, ANRLO:ANRIND)
+#'
+#' vs_anrind2 <- vs2
+#' derive_var_anrind(use_a1hia1lo = FALSE) %>%
 #'   select(USUBJID, PARAMCD, AVAL, ANRLO:ANRIND)
-derive_var_anrind <- function(dataset) {
-  assert_data_frame(dataset, required_vars = exprs(ANRLO, ANRHI, AVAL))
+#'
+#' vs_anrind1
+#' vs_anrind2
+derive_var_anrind <- function(dataset,
+                              use_a1hia1lo = TRUE) {
```

Comment:
looks like you have to still run `devtools::document()` and push the updated `.Rd` file to get rid of the warning, in addition if you continue to get the note:
![image](https://github.com/pharmaverse/admiral/assets/43018762/8534dd91-8e69-4664-865c-4bb5c731cc89)

add them to `R/globals.R`